src/libs/blueprint/conduit_blueprint_mesh.cpp

```diff
@@ -1133,8 +1178,9 @@ struct TopologyMetadata
         // associated conversations (data).
         Node temp, data;
 
-        // NOTE(JRC): A 'deque' is used in order to allow for queue behavior
-        // to be used in deciding the identifiers for the cascade of entities.
+        // NOTE(JRC): A 'deque' is used so that queue behavior (FIFO)
+        // is responsible for ordering the identifiers in the cascade of entities,
+        // which more closely follows the average intuition.
         const index_t bag_num_elems = topo_num_coords + topo_num_elems;
```

Comment:
I like this term.
(FYI: i am below average intuition :-) )

Reply:
I think the phrase I was looking for was "standard intuition," but calling this "average" (as opposed to enlightened forms such as LIFO, etc.) is more comical.